docs/source/pages/cli_usage/index.rst

```diff
@@ -0,0 +1,61 @@
+.. Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+.. _cli-usage:
+
+==================
+Command Line Usage
+==================
+
+You could use the bash script ``run_macaron.sh`` we provided to run Macaron as a Docker image (For more information on how to get this script, please see :ref:`Download Macaron <download-macaron>`).
```

Comment:
```suggestion
Use the bash script ``run_macaron.sh`` to run Macaron as a Docker container (for more information on how to get this script, please see :ref:`Download Macaron <download-macaron>`).
```

Reply:
Addressed in https://github.com/oracle-samples/macaron/pull/269/commits/602d68376381eb591484d8df2ecaee0e26dd0e7e